src/renderer/components/wallet/txs/send/Send.hooks.ts

```diff
@@ -4,14 +4,15 @@ import { Address } from '@xchainjs/xchain-client'
 import { Asset, assetToString } from '@xchainjs/xchain-util'
 import { useHistory } from 'react-router-dom'
 
+import { Network } from '../../../../../shared/api/types'
 import * as walletRoutes from '../../../../routes/wallet'
 
 export const useChangeAssetHandler = () => {
   const history = useHistory()
 
   const handler = useCallback(
-    (asset: Asset, walletAddress: Address) => {
-      const path = walletRoutes.send.path({ asset: assetToString(asset), walletAddress })
+    (asset: Asset, walletAddress: Address, network: Network) => {
```

Comment:
In this hook `network` is accessible via `AppContenxt` - no need to pass it around.